test/python/transpiler/test_mappers.py

```diff
@@ -0,0 +1,280 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Meta tests for mappers.
+
+The test checks the output of the swapper to a ground truth DAG (one for each
+test/swapper) saved in as a pickle (in `test/python/pickles/`). If they need
+to be regenerated, the DAG candidate is compiled and run in a simulator and
+the count is checked before being saved. This happens with (in the root
+directory):
+
+> python -m  test.python.transpiler.test_mappers
+
+To **add a swapper**: add a new class to this file:
+
+* inheriting from ``CommonTestCases, QiskitTestCase``
+* overwrite the required attribute ``pass_class``
+
+For example::
+
+    class TestsSomeSwap(CommonTestCases, QiskitTestCase):
```

Comment:
Non blocking, would not be simpler if `CommonTestCases` inherits from `QiskitTestCase`?

Reply:
Unittest treats the subclasses of `unittest.TestCase` in a particular way...